ciao-controller/instance.go

```diff
@@ -359,8 +359,8 @@ func newConfig(ctl *controller, wl *types.Workload, instanceID string, tenantID
 		userData.Hostname = "cnci-" + tenantID
 	}
 
-	// handle storage resources in workload definition
-	if wl.Storage != nil {
+	// if no instance volumes specified, use those in workload
+	if len(volumes) == 0 && wl.Storage != nil {
```

Comment:
Right, but what if the workload has two volumes, or two volumes are specified for the instance then it's confusing.

I feel like the behaviour needs to be predictable and explainable - which this is: instance volume specification always overrides workload specification.